e2e/windows/e2e-helper.sh

```diff
@@ -16,11 +16,23 @@ exec_on_host() {
     kubectl exec $(kubectl get pod -l app=debug -o jsonpath="{.items[0].metadata.name}") -- bash -c "nsenter -t 1 -m bash -c \"$1\"" > $2
 }
 
-create_storage_account() {
+create_storage_container() {
     set +x
-    az storage account create -n $WINDOWS_E2E_STORAGE_ACCOUNT -g $RESOURCE_GROUP_NAME -l $LOCATION
-    account_key=$(az storage account keys list --account-name $WINDOWS_E2E_STORAGE_ACCOUNT --query "[0].value")
-    az storage container create -n "tmp" --account-name $WINDOWS_E2E_STORAGE_ACCOUNT --account-key $account_key
+    # check if the storage container exists and create one if not
+    exists=$(az storage container exists --account-name $STORAGE_ACCOUNT_NAME --account-key $MAPPED_ACCOUNT_KEY --name $WINDOWS_E2E_STORAGE_CONTAINER)
```

Comment:
Will the cluster for the specified k8s version be kept for 3 days as the current design? Who will delete the container created here and when?

Reply:
The current design will reclaim the resource group including the cluster in about less than 1 day (fewer than 3 days) by my observation. @junjiezhang1997 is more familiar with this and maybe can clarify?

Reply:
The container will not be deleted. The current design is to clean up files in the container every time new cluster is created.

Reply:
Will the container be still there if the k8s version is not supported.

Reply:
It will I believe since it's stored in `akswinstorageaccount` in the vhd release sub that does not do cleaning. But I think it's out of the scope of this PR. Something like the backfill deletion I was and @junjiezhang1997 is working on?

Reply:
I think that you need to clean up it with adding the code in backfill deletion since these containers are created by this PR

Reply:
You have two options: 1. clean up the containers in this PR; 2. create a new task and do it in another pr

Reply:
Go with 2. Created a task to do backfill and commenting/formatting/documenting.